src/ast/passes/printer.cpp

```diff
@@ -26,7 +26,14 @@ std::string Printer::type(const SizedType &ty)
 void Printer::visit(Integer &integer)
 {
   std::string indent(depth_, ' ');
-  out_ << indent << "int: " << integer.n << type(integer.type) << std::endl;
+  out_ << indent << "int: " << integer.value << type(integer.type) << std::endl;
```

Comment:
```suggestion
  out_ << indent << "unsigned int: " << integer.value << type(integer.type) << std::endl;
``` ?

Reply:
Uh, I had this and had to update tons of change detector tests. I ended up reverting because I felt that the `int` case was by far more common and I didn't want to get into the weeds. WDYT? Should I make this universally `unsigned int`?

Reply:
I mean, this is just for AST debugging. I think it's better if we have a 'signed int' to have an 'unsigned int' but that could be a follow-up PR (or a "good first issue" issue).

Reply:
I'd rather leave this bit alone for this change. Certainly a valid clean-up later on, but it would also introduce a bunch of unrelated churn renamed `Integer` to `UnsignedInteger`. (And there's plenty of churn already.)

Reply:
Mind just making an issue for it?

Reply:
I will when it lands, if that's okay.

Reply:
Sure.